cupy/linalg/_eigenvalue.py

```diff
@@ -35,30 +39,57 @@ def _syevd(a, UPLO, with_eigen_vector):
     else:  # UPLO == 'U'
         uplo = cublas.CUBLAS_FILL_MODE_UPPER
 
-    if dtype == 'f':
-        buffer_size = cupy.cuda.cusolver.ssyevd_bufferSize
-        syevd = cupy.cuda.cusolver.ssyevd
-    elif dtype == 'd':
-        buffer_size = cupy.cuda.cusolver.dsyevd_bufferSize
-        syevd = cupy.cuda.cusolver.dsyevd
-    elif dtype == 'F':
-        buffer_size = cupy.cuda.cusolver.cheevd_bufferSize
-        syevd = cupy.cuda.cusolver.cheevd
-    elif dtype == 'D':
-        buffer_size = cupy.cuda.cusolver.zheevd_bufferSize
-        syevd = cupy.cuda.cusolver.zheevd
+    global _cuda_runtime_version
+    if _cuda_runtime_version < 0:
+        _cuda_runtime_version = runtime.runtimeGetVersion()
+
+    if (not runtime.is_hip and _cuda_runtime_version >= 11020):
```

Comment:
Oops @toslunar we missed this line...Could you send a PR to change it? 😅

Reply:
Thanks!  Created PR #6201.

Reply:
oops, Thanks.